### DISCUSSION TITLE
[SOLVED]How can i remove a mirrored profile

### DISCUSSION THREAD

**Discussion by IOTAkn0wl3dg3** [iotaledger/firefly#2226] [Help & Support] [answered]
Hi, i had some issues with synching in firefly wallet, so i created a new profile and restored from ledger. Now everything works fine.
i want to delete the 2nd profile cause i need only one. I can't delete it cause the messages appears you have already funds on it. 
Is there any tweak to delete a 2nd identical profile? i tried to deinstall firefly but the profile is still on my machine after reinstallation. 
Windows 10 is the OS.  Many thanks.

**Comment by t00k** [comment, marked as answer]
Which operating system are you using?

**Reply by IOTAkn0wl3dg3** [reply]
Windows 10, i updated at the moment the post ;)

**Reply by t00k** [reply]
C:\Users\Username\AppData\Roaming\Firefly\__storage__
In that folder you will see your different profiles as folders. Delete the one you want to remove.
Please make sure you have your profiles securely backed up before you delete the folder though.

To browse quickly into AppData you can browse to %APPDATA%

**Reply by IOTAkn0wl3dg3** [reply]
thanks good information, i take look. yes i know the environment variable. im working on windows systems since 20 years ;)
i  have ledger nano, so i have my backup.

**Reply by t00k** [reply]
Awesome, then you'll find it easy enough and be able to remove it.
Stay safe out there, a lot of scammers in crypto!

**Reply by IOTAkn0wl3dg3** [reply]
i'm very careful with information, thank you for support and fast reply t00k!

**Reply by IOTAkn0wl3dg3** [reply]
worked
